src/darwin/Framework/CHIP/templates/availability.yaml

```diff
@@ -8823,6 +8827,52 @@
               - AttributeList
               - FeatureMap
               - ClusterRevision
+          MicrowaveOvenControl:
+              - CookTime
+              - MaxCookTime
+              - PowerSetting
+              - MinPower
+              - MaxPower
+              - PowerStep
+              # SupportedWatts and SelectedWattIndex are Provisional in Matter 1.3
```

Comment:
Then they need explicit provisional annotations, right?

Reply:
Yep - is [this](https://github.com/project-chip/connectedhomeip/pull/33399/files#diff-94ddfb24d1680d06578d1e18f05664162d7c7e9f05958813917cde261c7e9541R9765-R9767) correct?

Reply:
Generally, yes, but I would add a comment explaining why those are marked provisional.